### ISSUE TITLE
UI Translate code removes same-level content

### ISSUE THREAD

**Issue by richardlawley** [apache/cloudstack#3385]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.11.2
~~~

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

N/A

##### SUMMARY

The UI code replaces `<translate key="blah"/>` with the localised content.  However, the translate code incorrectly removes content at the same level, for example the Zone Names on the capacity list of the dashboard:

```
<div class="label">
    <translate key="label.zone"/>: <span data-list-item="zoneName"></span>
</div>
```

This results in the dashboard not showing the Zone name.  This also affects the "Optional" label in VM creation from a VPC on the Network step.

This is because the `$.find('translate[key]')` code is not treating translate as a valid self-closing tag.  This problem doesn't affect cases where there is no non-tag content at the same level, which is why it hasn't been picked up yet.

Other known cases:
* Instance Wizard - VPC Select Network tier - optional label not shown
* Instance Wizard - Step 8 (Review) - Name and Group do not show Optional Label

##### STEPS TO REPRODUCE

View the Dashboard
Observe that the Capacities do not list zone names

##### EXPECTED RESULTS

Zone Names should be displayed in Capacity list

##### ACTUAL RESULTS

![image](https://user-images.githubusercontent.com/1608083/59103002-8f26f100-8925-11e9-87d3-9a4344ca7523.png)

**Comment by anuragaw** [comment]
This should be fixed on latest master @richardlawley https://github.com/apache/cloudstack/pull/3326

CC/ @rhtyd , @rafaelweingartner , @PaulAngus

**Comment by richardlawley** [comment]
@anuragaw #3326 only fixes the Dashboard instance - there were about 10 occurrences of the same problem which I fixed in #3386.  However, the method you used to fix (wrapping in a label) is better than mine.  Will try and alter mine to fix the other occurrences.
